demos/code/Blocks/Chapter 5 - Functions.js

```diff
@@ -765,6 +602,480 @@ Blockly.C['func_return'] = function(block) {
 	return code;
 };
 
+Blockly.Blocks['func_parameters_call'] = {
```

Comment:
This is just a rename of the block. The reference in index.html has been updated as well.